src/Culling/babylon.boundingBox.ts

```diff
@@ -93,7 +96,7 @@
             max.addToRef(min, this.center).scaleInPlace(0.5);
             max.subtractToRef(min, this.extendSize).scaleInPlace(0.5);
 
-            this._update(worldMatrix || this._worldMatrix || Matrix.Identity());
+            this._update(worldMatrix || _identityMatrix);

```

Comment:
What if I call it from the outside with reConstruct(min, max)? It will use identity instead of the current wm, right? This seems like an unwanted side effect